### PR TITLE
Adding @puerco to streering and admin groups

### LINKED ISSUE #1265: [SC] Add new member Adolfo to steering committee
The following tasks need to be done 

- Add @puerco as new member
- Remove @itsmurugappan 

- [x] Add new member in [STEERING-COMMITTEE.md](https://github.com/knative/community/blob/main/STEERING-COMMITTEE.md)
- [x] [peribolos](https://github.com/knative/community/tree/main/peribolos) for both knative and knative-sandbox 
- [x] groups for mailing lists
  - https://github.com/knative/community/blob/main/groups/committee-steering/groups.yaml
  - https://github.com/knative/community/blob/main/groups/groups.yaml#L60
- [x] [CNCF Project maintainers](https://github.com/cncf/foundation/blob/main/project-maintainers.csv)
- [x] CNCF Knative Project JIRA access (via [cncf service desk](https://cncfservicedesk.atlassian.net/servicedesk/customer/portal/1) Jira ticket)
- [x] CNCF Knative maintainers group list (via [cncf service desk](https://cncfservicedesk.atlassian.net/servicedesk/customer/portal/1) Jira ticket)
- [x] New Blog post welcoming recent members @nainaz @salaboy @puerco to Steering Committee (Use [Example](https://knative.dev/blog/steering/2021-12-14-steering-elections-results/))

### PR DESCRIPTION
**Pull request by salaboy**
# Changes

- :gift: Adding @puerco to steering and admin

/kind enhancement 

Fixes #1265

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

**Comment by dprotaso** [comment]
you need to run `hack/update-codegen`

File: peribolos/knative.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ orgs:
     - evankanderson
     - google-admin
     - googlebot
-    - itsmurugappan
```
Comment (dprotaso) [inline peribolos/knative.yaml:18]: Please move `itsmurugappan` to the member list

**Comment by dprotaso** [comment]
bump @salaboy 

cc @knative/steering-committee

File: peribolos/knative.yaml [hunk 2]
```diff
@@ -500,7 +501,7 @@ orgs:
             description: Knative Steering Committee
             maintainers:
             - csantanapr
-            - itsmurugappan
+            - puerco
```
Comment (puerco) [inline peribolos/knative.yaml:504]: @salaboy  I think these lists need to be sorted (I tried to push a commit to you r branch but I can't)

**Comment by puerco** [comment]
@salaboy : I've opened https://github.com/knative/community/pull/1316 to fix my affiliation and added a few commits to fix the peribolos config, superseding this PR .

**Pull request closed without merging**